.github/workflows/build.yml

```diff
@@ -202,6 +202,8 @@ jobs:
 
       - name: Find and check all scripts using ShellCheck
         uses: ludeeus/action-shellcheck@master
+        with:
+          ignore_names: gradlew
```

Comment:
This should be part of the previous commit, I think.

Reply:
I agree but I got lazy and wanted to avoid rewriting history. Will do!